srcpkgs/gprbuild/template

```diff
@@ -0,0 +1,63 @@
+# Template file for 'gprbuild'
+pkgname=gprbuild
+version=20180528
+revision=1
+create_wrksrc=yes
+only_for_archs="i686 x86_64 x86_64-musl"
+makedepends="gcc-ada libada-devel"
+short_desc="Advanced build system"
+maintainer="John Hamilton <hamilton@disroot.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/AdaCore/gprbuild"
+distfiles="http://mirrors.cdn.adacore.com/art/5b0819dfc7a447df26c27a68>gprbuild-gpl-2018-20180524-src.tar.gz
+	http://mirrors.cdn.adacore.com/art/5b0819dec7a447df26c27a40>xmlada-gpl-2018-20180524-src.tar.gz"
+checksum="a06b7ee6ddc00a927db9b6207a4bd44e670ff2cceb9879a5baa98f4a38f552a1
+	aacef3ae91e701690f60bde9c8a4a3f47ba8ca1f8f3c0599d01bce0173be80de"
+
+_bin_progs="gprbuild gprconfig gprclean gprinstall gprname gprls"
+_lib_progs="gprlib gprbind"
+
+do_build() {
+	cd gprbuild-gpl-2018-src
+	patch -Np0 -i ${FILESDIR}/gprbuild-2018.patch
+
+	srcdir=$PWD
+	xmlada_src=../xmlada-gpl-2018-src
+
+	CC=${CC:-cc}
+	GNATMAKE=${GNATMAKE:-gnatmake}
+	CFLAGS=${CFLAGS:-$CFLAGS}
+	GNATMAKEFLAGS=${GNATMAKEFLAGS:--j0}
+
```

Comment:
Why not use XBPS_MAKEJOBS here?

Reply:
XBPS_MAKEJOBS is just a number, you want the `makejobs` variable